paper-input-behavior.html

```diff
@@ -85,15 +85,6 @@
       },
 
       /**
-       * Set to true to prevent the user from entering invalid input. If you're
-       * using PaperInputBehavior to  implement your own paper-input-like element,
-       * bind this to `<input is="iron-input">`'s `preventInvalidInput` property.
-       */
-      preventInvalidInput: {
```

Comment:
They were both either used together, or not at all. (like, you can't just use one without the other). Did I misunderstand it?